gdal/frmts/georaster/oci_wrapper.cpp

```diff
@@ -238,6 +237,21 @@ OWConnection::OWConnection( const char* pszUserIn,
 
     bSuceeeded = true;
 
+    // ------------------------------------------------------
+    //  If no user specified, get it from the session 
+    // ------------------------------------------------------
+    if (EQUAL(pszUser, "") )
+    {
+      OWStatement* poStmt = CreateStatement(
+            "select sys_context('userenv','session_user')\n"
+            "from dual\n" );
+
+      poStmt->Define(pszUser);
+      poStmt->Execute();
+      delete poStmt;
```

Comment:
shouldn't the return of Execute() be tested ?

Reply:
Sure. I will fix this.